### PR TITLE
Install lc0 on openSUSE - Full documentation, Install script and links to experimental RPM packages

### PR DESCRIPTION
**Pull request by putztzu**
1. Modifies README.md
Link to openSUSE_install.md

2. New file openSUSE_install.md
Instructions for installing on openSUSE including link to install_openSUSE_lco.sh, links to experimental RPM packages and instructions for setting up with Arena.

3. New file install_openSUSE_lc0.sh
Fully unattended install script supporting all current, past and future versions of openSUSE LEAP and Tumbleweed releases.s The User executes the script and waits a few minutes for completion. Fully commented, the script should be easy to read.

### CODE REVIEW

**Comment by putztzu** [comment]
Hello, don't understand the Appveyor issue that is failing. The proposed change to README.md references a new file which is included in the Pull Request so I believe there are no missing parts. And of course, it all works now in my own cloned copy of the lc0 github project.

**Comment by mooskagh** [comment]
You modified README.md, but meanwhile someone also touched that file.
You have to merge it before proceeding.
Either merge changes from master branch or rebase this PR on latest master.

**Comment by putztzu** [comment]
Thx, will look into that.

File: openSUSE_install.md~ [hunk 1]
```diff
@@ -0,0 +1,218 @@
+![openSUSE Logo](https://github.com/openSUSE/artwork/blob/master/Logo/official-logo-color.png?style=centerme)
```
Comment (mooskagh) [inline openSUSE_install.md~]: That's some editor temporary file, I believe it should not be in the PR.

Comment (putztzu) [reply openSUSE_install.md~]: Hi,
If an image link violates project policy or could somehow be made to be a security risk, I can certainly replace with text.
If an image link cannot be considered a security risk, I thought an image could enhance appearance.

Comment (mooskagh) [reply openSUSE_install.md~]: I meant `openSUSE_install.md~` which seems to be a temporary file for `openSUSE_install.md`. I believe only one of them (without `~`) has so be checked in.

Comment (putztzu) [reply openSUSE_install.md~]: Thank you for your clarification, I removed all temporary files that were created by text editors.
Should be good to go now unless there is anything else.

**Pull request merged**